tests/check_framework/urls/contains_tuple.py

```diff
@@ -0,0 +1,7 @@
+from django.conf.urls import include, url
+
+urlpatterns = [
+    url('^', include([
```

Comment:
Sure, if you just want to remove the include here for now I think that would be fine.